Jenkinsfile

```diff
@@ -225,6 +154,7 @@ pipeline {
                             skopeo delete \
                               --creds ${env.QUAY_CREDS} \
                               docker://${env.OPERATOR_CONTAINER_IMAGE_CANDIDATE_NAME} \
+                            || sleep 10
```

Comment:
Deletion of recently pushed tag sometimes fails. Giving it some pause before another retry helps.